1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -16,8 +18,42 @@ const BIRTHDAYS = [
     "November 15th"
 ];
 
+console.log();
+
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  // TODO
+// lets sort the array first a-z
+  
+  birthdays.sort();
+  let i = 0;
+  while (i < birthdays.length) {
+    if (birthdays[i].includes('July')) {
+      return birthdays[i]
```

Comment:
Works, good work, bit much commnented out code here I would say See: https://syllabus.codeyourfuture.io/guides/code-style-guide#dont-leave-lots-of-commented-out-code